swagger-generator/src/main/java/com/mrv/yangtools/codegen/impl/AbstractDataObjectBuilder.java

```diff
@@ -254,9 +256,13 @@ protected Pair prop(DataSchemaNode node) {
             prop = refOrStructure((ContainerSchemaNode) node);
         } else if (node instanceof ListSchemaNode) {
             prop = new ArrayProperty().items(refOrStructure((ListSchemaNode) node));
-        } else if (node instanceof AnyXmlSchemaNode) {
+        } else if (node instanceof AnyxmlSchemaNode) {
             log.warn("generating swagger string property for any schema type for {}", node.getQName());
             prop = new StringProperty();
+        } else if (node instanceof InputSchemaNode) {
```

Comment:
Why do we need that?
If this is a fix for an existing bug in processing I would prefer to have it resolved as a separate issue  + provide test that covers the problem that got fixed